user-console/src/main/resources/org/pentaho/mantle/public/themes/ruby/mantleRuby.css

```diff
@@ -1270,42 +1267,6 @@ div#customDropdownPopupMajor .gwt-MenuBar-vertical .gwt-MenuItem:hover:not(.gwt-
 .toolbar-button-disabled img {
   cursor: default !important;
 }
-/* endregion Main Toolbar */
-
-.schedulesPanel .toolbar {
-  background: none;
-  border: 0;
-  margin-bottom: 5px;
-}
-
-.schedulesPanel .pentaho-table th {
-  font-size: 12px;
-  border-spacing: 0;
-  color: #FFF;
-  padding: 8px 10px 10px 10px;
-  font-weight: normal;
-}
```

Comment:
These should be kept here. See related comment: https://github.com/pentaho/pentaho-commons-gwt-modules/pull/877#discussion_r1152113875.

Reply:
Added back.